feat: add folder exclusions (#54)
lacework/terraform-gcp-config: examples/organization-level-config-exclude-folders/README.md

# Integrate a Google Cloud Organization with Lacework for Configuration Assessment
The following provides an example of integrating an entire Google Cloud Organization with Lacework for Cloud Resource configuration assessment.

The fields required for this example are:

| Name | Description | Type |
|------|-------------|------|
| `org_integration` | Set this to `true` to configure an organization level integration. | `bool` |
| `organization_id` | The id of the GCP Organization to integrate with. | `string` |
| `project_id` | The id of a Project, which will be used to deploy required resources for the integration. Note: if this is var is not explicitly set, the provider will check for the presence of the `GOOGLE_PROJECT` env var | `string` |

> | `exclude_folders` | Enables logic to exclude a list of folders from the integration. Default is false | `bool` |

Reviewer: ```suggestion
```

> | `folders_to_exclude` | List of root folders to exclude if `exclude_folders` is set to `true`.  Format is 'folders/1234567890' | `set(string)` |

Reviewer: ```suggestion
| `folders_to_exclude` | List of root folders to exclude in an organization-level integration.  Format is 'folders/1234567890' | `set(string)` |
```
